pyccl/halos/profiles_2pt.py

```diff
@@ -102,15 +103,13 @@ class Profile2ptHOD(Profile2pt):
     where all quantities are described in the documentation of
     :class:`~pyccl.halos.profiles.HaloProfileHOD`.
     """
-    def fourier_2pt(self, prof, cosmo, k, M, a,
-                    prof2=None, mass_def=None):
+
+    @warn_api
+    def fourier_2pt(self, cosmo, k, M, a, prof, *, prof2=None, mass_def):
         """ Returns the Fourier-space two-point moment for the HOD
         profile.
 
         Args:
-            prof (:class:`~pyccl.halos.profiles.HaloProfileHOD`):
-                halo profile for which the second-order moment
-                is desired.
```

Comment:
I think you forgot to copy this below

Reply:
yep - I was going to go through the docs either way, but good call